### DISCUSSION TITLE
Pedir ajuda é diferente de pedir para resolverem seu problema

### DISCUSSION THREAD

**Discussion by kvnol** [frontendbr/forum#526] [Discussão]
Pessoal, ontem fiz um post no Facebook (que logo [migrei para o Medium](https://medium.com/@kvnol/pedir-ajuda-%C3%A9-diferente-de-pedir-para-resolverem-seu-problema-49b3abebebca)) sobre o chorume nos grupos de desenvolvimento web, onde a galera que está iniciando como dev não faz o mínimo esforço para fazer alguma coisa funcionar e já vai logo jogando blocos de código nos grupos pedindo, basicamente, para alguém resolver aquele problema.

> “Oi, galera! Eu queria saber como eu posso ativar uma classe só SE eu clicasse no bot…” — “se”? Eu acho que já ouvi falar disso…
> -> [Ler mais no Medium](https://medium.com/@kvnol/pedir-ajuda-%C3%A9-diferente-de-pedir-para-resolverem-seu-problema-49b3abebebca)

Se alguém tiver uma opinião contrária à minha ou algo a acrescentar, vamos lá!

**Comment by fititnt** [comment]
> Não sei vocês, mas a sensação orgasmática de resolver um problema sozinho sem copiar nenhuma linha de código é incrível. Pode ser a coisa mais besta, mas quem nunca resolveu algum problema sozinho e gritou “Aê p%R$@, consegui resolver!”. Eu recomendo a vocês sentirem esse prazer pelo menos uma vez por dia.

@kvnol uma base científica para o que está falando aqui é [metacognição](http://www.psicologiaexplica.com.br/o-que-e-metacognicao/) e também vale a pena saber mais sobre **piramide de aprendizagem**.

Infelizmente no Brasil não ensinam crianças desde cedo a aprender a aprender. Algumas ações que "novatos" fazem em grupos não é apenas questão de desconhecimento com interações sociais, mas em parte o cérebro humano funciona melhor com ajuda construtiva.

Quem sabe mais desses pontos de base consegue direcionar com eficiência extremamente alta uma pessoa a conseguir aprender/fixar conhecimento, mesmo usando frases curtas e incentivadoras.

**Comment by kvnol** [comment]
@fititnt Que coisa linda isso, cara! Exatamente como você disse, muitas vezes até eu tenho problema em aprender algo e como organizar minhas ideias para aprender algo (ainda mais sozinho). Sempre que chego neste ponto, eu travo e sou obrigado a buscar algum professor, ou qualquer pessoa mais experiente, para que eu possa ter auxílio na **organização do aprendizado** (em qualquer aprendizado, seja desenvolvimento, música, exercícios físicos e outros).

Eu sou um cara que preciso estudar mais o "learn to learn".
